frontend/src/components/SearchBar.tsx

```diff
@@ -0,0 +1,37 @@
+import MultiselectDropdown from "./MultiselectDropdown.tsx";
+import TextInput from "./TextInput.tsx";
+import Button from "./Button.tsx";
+import {useState} from "react";
+
+function SearchBar(){
+    const categoryOptions = ['Pets', 'Home', 'Food', 'Cleaning'];
```

Comment:
Since these values ​​will always be the same, that is, they will be like constants, we can move them outside the component to prevent them from being recreated with each render. In addition, we can mark it as `const` so that the values ​​also become `readonly`.

```suggestion
    const categoryOptions = ['Pets', 'Home', 'Food', 'Cleaning'] as const;
    
    function SearchBar(){
```

We can do the same with `stockOptions` and then we could also move `categoryOptions` and `stockOptions` to a dedicated constants file, since keeping them inside the component could cause a problem with `cyclic references` in the future if these variables are used by other components.